### ISSUE TITLE
Turn off remote validation

### ISSUE THREAD

**Issue by AdrianB93** [proengsoft/laravel-jsvalidation#145]
Hello,

In my application, `disable_remote_validation` does not work. I have is set to `true` in `config/jsvalidation.php` because "Yes, I want it disabled" and it is still attempting to do remote validation, I've actually had to hack around this issue in my controllers by exploding the validation strings for edit and create views and removing all rules I wish to ignore for the client side validation.

I do not want to do remote validation, I only want basic requires and min/max fields, required_unless, etc validations to work. I don't want file sizes or types to be checked, nor do I want my custom rules checked, in those cases I'm fine with being redirected back to the form to list errors.

Regardless of whether this `disable_remote_validation` config setting is true or false, it still makes requests, I observed this in the console logs in Google Chrome. Usually I either get a whoops error on a form field with custom validation, or the form simply will not submit.

I'm not saying I wouldn't necessarily like to use remote validation in the future, it just hasn't worked initially and I don't want to use it until I have time to look further into how to make it work. This is why I need the `disable_remote_validation` config setting to work.

Cheers.

**Comment by AdrianB93** [comment]
Not really sure if my solution is worth a pull request, I've forked the project and put what I'm doing to get the client side validation working. I made an array of rule names that I know don't (or at least shouldn't) cause remote validation to occur and filter them out of the rules array passed into JsValidatorFactory.

After looking over the project, unless I'm mistaken `required_unless` should be a client side validation but it's still triggering a remote request which causes my form's submit button to stop working. No Javascript errors appear in the console.

Put simply `disable_remote_validation` is not working, I couldn't figure out why but worked around it in this commit: https://github.com/AdrianB93/laravel-jsvalidation/commit/2fa6554fc80cb0700e9796651fdd6df28067fe04

**Comment by torrentalle** [comment]
`required_unless` Javascript validation is implemented in **master** branch, not in latest stable. For this reason is validated using AJAX.
If `disable_remote_validation` is **true**, `required_unless`should't be validated. I will fix in next release

**Comment by torrentalle** [comment]
Fixed in latest version
